README.md

```diff
@@ -1,51 +1,230 @@
+- Finally, click the green `Create Pull Request` button. Great job! You did it!
+
+You can ask questions by raising an [issue](https://github.com/EddieHubCommunity/open-source-practice/issues/new).
+
+### Option 2. Complete this process on your computer (locally)
+
+**1. Fork the project:**
+
+- Click the gray Fork button at the top right of this page. This creates your copy of the project and saves it as a new repository in your GitHub account. ![fork](https://github.com/AmanxUpadhyay/open-source-practice/assets/76415079/00b41de9-768c-4a01-a27e-f7aa75cd6a27)
+
+**2. Clone this project on your computer:**
+
+- Go to your profile. You will find forked repo named **_open-source-practice_**. go to the repo by clicking on it.
+- Click on the green Code button, then either the HTTPS or SSH option, and, click the icon to copy the URL. Now you have a copy of the project. Thus, you can play around with it locally on your computer.
+
+- Run the following commands into a terminal window (Command Prompt, Powershell, Terminal, Bash, ZSH). Do this to download the forked copy of this repository to your computer. ![git-clone](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/4d600e25-83b1-4e8f-9325-f1adc4f8ce3d)
+
+```bash
+  git clone https://github.com/YOUR_GITHUB_USERNAME/hacktoberfest-practice.git
+```
 
-* Add folder.
-* Commit and push your code.
+- Switch to the cloned folder. You can paste this command into the same terminal window. ![opening-vscode](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/1a8b350d-0c96-461a-85f4-a59185aed6b6)
 
-```markdown
-git add .
-git commit -m 'Commit message'
-git push origin branch-name
+```bash
+  cd hacktoberfest-practice
 ```
 
-* Create a new pull request from your forked repository (Click the `New Pull Request` button located at the top of your repo)
-* Wait for your PR review and merge approval...
-* __Star this repository__ if you had fun contributing!
+**3. Create a new branch:**
+
+- Your username would make a good branch because it's unique. ![git-checkout](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/68ab0380-c731-4e67-bccc-b666da5dd174)
+
+```bash
+  git checkout -b <name-of-new-branch>
+```
+
+**4. Edit:**
+
+- Open the `README.md` file
+
+- **Add your name to the section that matches your Initial in [this list](https://github.com/EddieHubCommunity/open-source-practice#hacktoberfest-community), make sure that your name is in alphabetical order. Then save your changes.**
+
+- For example
+  `- [Full Name](https://github.com/your-username)`
+
+**5. Stage your changes:**
+
+```bash
+  git add README.md
+```
+
+or
+
+```bash
+  git add .
+```
+
+**6. Commit the changes:**
+![git-commit](https://github.com/AmanxUpadhyay/hacktoberfest-practice/assets/76415079/9a6a58f3-ed0b-4bc2-b749-48baede77835)
+
+```bash
+  git commit -m "Add <your-github-username>"
```

Comment:
instead of GitHub username, it should be "meaningful comment"

Reply:
doen